store-pg/src/main/scala/com/socrata/pg/store/PGSecondary.scala

```diff
@@ -53,14 +53,8 @@ class PGSecondary(val config: Config) extends Secondary[SoQLType, SoQLValue] wit
     client
   }
 
-  val rowsChangedPreviewConfig = curator match {
-    case Some(c) =>
-      val config = new RowsChangedPreviewConfig.ZKClojure(c, "/soql-postgres-secondary")
-      config.start()
-      config
-    case None =>
-      RowsChangedPreviewConfig.Default
-  }
+  val rowsChangedPreviewConfig =
+    new RowsChangedPreviewConfig.Clojure(storeConfig.transparentCopyFunction.getOrElse(RowsChangedPreviewConfig.default))
```

Comment:
Not a clojure expert, but `transparentCopyFunction` and `default` look like same function?

Reply:
They do and are!  But we can change transparentCopyFunction in apps_marathon.